app/gui/view/graph-editor/src/lib.rs

```diff
@@ -2819,9 +2830,15 @@ fn new_graph_editor(app: &Application) -> GraphEditor {
 
         input_add_node_way <- inputs.add_node.constant(WayOfCreatingNode::AddNodeEvent);
         input_start_creation_way <- inputs.start_node_creation.constant(WayOfCreatingNode::StartCreationEvent);
+        input_start_node_creation_from_port <- inputs.hover_node_output.sample(&inputs.start_node_creation_from_port);
+        start_node_creation_from_port <- input_start_node_creation_from_port.filter_map(|v| v.clone());
+        start_creation_from_port_way <- start_node_creation_from_port.map(
+            |endpoint| WayOfCreatingNode::StartCreationFromPortEvent{endpoint: endpoint.clone()});
+        removed_edges_on_node_creation_from_port <= start_creation_from_port_way.map(f_!(model.model.clear_all_detached_edges()));
+        out.source.on_edge_drop <+ removed_edges_on_node_creation_from_port;
```

Comment:
1. Some of those lines are over 100 characters.
2. In this section, FRP nodes are for just creating the `way` structure. Please move nodes specific to creating node from clicking output to a separate block.